tests/test_views.py

```diff
@@ -155,13 +157,15 @@ def test_invalid_messages(self, mock_role, mock_waffle, mock_render):
     @patch('learning_assistant.views.get_user_role')
     @patch('learning_assistant.views.CourseEnrollment.get_enrollment')
     @patch('learning_assistant.views.CourseMode')
-    def test_chat_response(self, mock_mode, mock_enrollment, mock_role, mock_waffle, mock_chat_response, mock_render):
+    def test_chat_response_default(
```

Comment:
Question: are there any error cases/non-happy paths that should be tested?

Reply:
humm yeah the only input we're adding is the query param, bad data for everything else is already tested above. I think I should validate an invalid variation doesn't do anything unexpected tho.

Reply:
okay, added a case for an invalid variation and noticed we needed better coverage on what template gets loaded since the way this was mocked left some holes